requirements-docs.txt

```diff
@@ -1,6 +1,5 @@
 # All pip installable requirements pinned for readthedocs.org
 fabric==1.10.0
-pystache==0.5.4
```

Comment:
This reminded me, that we don't test `html` plugin automatically. I double checked by running coverage and yes, unless one uses `make link` it's not involved.

On the other hand we test it manually before release, but I think it ought to be added to travis check, even if only as `make link && make smokecheck` after each check.